tests/missing-placement.test.html

```diff
@@ -0,0 +1,36 @@
+<html>
+  <body>
+    <div data-ea-publisher="test" data-ea-manual="true"></div>
+
+    <script type="module">
+      import { expect } from "@open-wc/testing";
+      import { runTests } from "@web/test-runner-mocha";
+      import { default as sinon } from "sinon";
+
+      import { wait, load_placements, Placement } from "../index";
+
+      // Don't actually call the server
+      // https://sinonjs.org/releases/v17/stubs/
+      let stub = sinon.stub(Placement.prototype, "fetch");
+      const response_html =
+        "<div><!-- A real ad would be here normally --></div>";
+      const elem_placement = document.createElement("div");
+      elem_placement.innerHTML = response_html;
+      stub.resolves(elem_placement.firstChild);
+
+      runTests(async () => {
+        describe("EthicalAds library", () => {
+          it("skips manual placements", async () => {
+            const placements = await wait;
+            expect(placements).to.be.an("array").that.includes(null);
+          });
+          it("can force load manual placements", async () => {
+            const placements = await load_placements(true);
+            expect(placements).to.be.an("array");
+            expect(placements[0]).to.have.property("publisher", "test");
```

Comment:
This is a funny syntax -- almost feels like Ruby.

Reply:
It is definitely heavily inspired by Rspec, Jest is too to some degree.

I should have pointed out that Chai is most commonly used with BDD style syntax, but there are a few options in the library:

https://www.chaijs.com/guide/styles/

The assert API might feel more at home, and is completely okay to use instead of expect/should.

Reply:
Yea, that assert style is definitely more akin to Python testing. I don't feel too strongly though, but it's nice they've provided both syntaxes for various backgrounds :)